src/main/java/javax/validation/Configuration.java

```diff
@@ -254,6 +269,17 @@
 	ClockProvider getDefaultClockProvider();
 
 	/**
+	 * Returns the default {@link ValueExtractor} implementations as per the
+	 * specification.
+	 *
+	 * @return the default {@code ValueExtractor} implementations compliant
+	 * with the specification
+	 *
+	 * @since 2.0
+	 */
+	Set<ValueExtractor<?>> getDefaultValueExtractors();
```

Comment:
Yes, consistency is good. But do you see any actual scenario where one would use them?